### PR TITLE
Cs 3718 inline spotlight

### PR DESCRIPTION
**Pull request by karloscastaneda**

### CODE REVIEW

File: static/inline-spotlight/scss/vendor/simple-scrollbar.scss [hunk 1]
```diff
@@ -0,0 +1,47 @@
+.ss-wrapper {
```
Comment (rfornes) [inline static/inline-spotlight/scss/vendor/simple-scrollbar.scss]: Why do we need the scrollbar?

File: static/inline-spotlight/js/vendor/jquery.ellipsis.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
+/*! jQuery ellipsis - v1.1.1 - 2014-02-23
+* https://github.com/STAR-ZERO/jquery-ellipsis
+* Copyright (c) 2014 Kenji Abe; Licensed MIT */
```
Comment (rfornes) [inline static/inline-spotlight/js/vendor/jquery.ellipsis.js]: Why do we need this? wasn't `trimText` doing the job?

File: static/inline/scss/styles.scss [hunk 3]
```diff
@@ -587,10 +587,7 @@ a {
 
 .tvp-video-item-active{
     .tvp-video-play{
-        background-color: #e57211;
-    }
```
Comment (rfornes) [inline static/inline/scss/styles.scss]: Why are hiding this?

File: static/inline-spotlight/scss/vendor/simple-scrollbar.scss (hunk 1 above)
Comment (karloscastaneda) [inline static/inline-spotlight/scss/vendor/simple-scrollbar.scss]: nope, will remove it

File: static/inline-spotlight/js/vendor/jquery.ellipsis.js (hunk 2 above)
Comment (karloscastaneda) [inline static/inline-spotlight/js/vendor/jquery.ellipsis.js]: hmm yeah , but the `trimText` it's a little bit different, it cuts text and the ellispsis adapts the contents to the container.

File: static/inline/scss/styles.scss (hunk 3 above)
Comment (karloscastaneda) [inline static/inline/scss/styles.scss]: we hide the 'Play icon' in order to show the 'Now Playing' text on the active video

**Pull request closed without merging**